### PR TITLE
chore(currencies): added latam fiat currencies

### PR DESCRIPTION
**Pull request by gusparr28**

### CODE REVIEW

**Comment by gusparr28** [comment]
closes https://github.com/fiatconnect/specification/issues/42

missing Latin American fiat currencies were added to FiatType https://github.com/fiatconnect/specification/blob/main/swagger.yaml#L39

File: swagger.yaml [hunk 1]
```diff
@@ -55,6 +55,19 @@ definitions:
       - RWF
       - CNY
       - XAF
+      - ARS
```
Comment (cajubelt) [inline swagger.yaml:58]: thanks, can you also add these to the FiatTypeEnum in the `fiatconnect-api.md` file?

Comment (gusparr28) [reply swagger.yaml:58]: thanks for the review. Sure, already done

**Review by cajubelt — APPROVED** [review]
thanks!

**Pull request merged**